tapdb/sqlc/queries/universe.sql

```diff
@@ -329,8 +327,8 @@ SELECT
     SUM(CASE WHEN event_type = 'SYNC' THEN 1 ELSE 0 END) AS sync_events,
     SUM(CASE WHEN event_type = 'NEW_PROOF' THEN 1 ELSE 0 END) AS new_proof_events
 FROM universe_events
-WHERE event_type IN ('SYNC', 'NEW_PROOF') AND
-      event_timestamp >= @start_time AND event_timestamp <= @end_time
+WHERE event_type IN ('SYNC', 'NEW_PROOF') 
+      AND event_timestamp BETWEEN @start_time AND @end_time
```

Comment:
:+1: makes sense. A comment that `BETWEEN` is inclusive for both start and end value might be helpful, as I assume the more explicit form was used to avoid needing a documentation lookup.